README.md

```diff
@@ -1,2 +1,109 @@
-# superheroes-automation
-automation for running quarkus superheroes benchmark load tests
+# Superheroes Load Test Automation
+Automation for running quarkus superheroes benchmark load tests
+
+## Prerequisites
+
+### QDup
+To run these scripts you need [qDup](https://github.com/Hyperfoil/qDup) that you can download directly from the GitHub repository:
+
+```bash
+wget https://github.com/Hyperfoil/qDup/releases/download/qDup-0.8.5/qDup-0.8.5-uber.jar
+```
```

Comment:
qDup can be run via jbang as well, it is in the Hyperfoil catalog. You can run it with 

`jbang qdup@hyperfoil http://example.com/scripts/qdup.yml`

Reply:
You are right, I can change to fully rely on jbang for this automation (I will give it a try first)!

Reply:
Updated the README